netCDF4/_netCDF4.pyx

```diff
@@ -1235,6 +1235,11 @@ is_native_big = numpy.dtype('>f4').byteorder == '='
 # hard code these here, instead of importing from netcdf.h
 # so it will compile with versions <= 4.2.
 NC_DISKLESS = 0x0008
+# introduced in 4.6.2
+if __netcdf4libversion__[0:5] >= "4.6.2":
+    NC_PERSIST = 0x4000
+else:  # prior to 4.6.2 this flag doesn't work, so make the same as NC_DISKLESS
+    NC_PERSIST = 0x0008
```

Comment:
So why not replace hard-coded `0x0008` with `NC_DISKLESS`.

Reply:
Good point - done